toolkit/tools/imagegen/configuration/configuration_test.go

```diff
@@ -456,14 +456,18 @@ var expectedConfiguration Config = Config{
 			},
 			PackageRepos: []PackageRepo{
 				{
-					Name:    "repo1",
-					BaseUrl: "https://repo1.com",
-					Install: false,
-				},
-				{
-					Name:    "repo2",
-					BaseUrl: "https://repo2.com",
-					Install: false,
+					Name:     "repo1",
+					BaseUrl:  "https://repo1.com",
+					Install:  true,
+					GPGCheck: true,
+					GPGKeys:  "file:///etc/pki/rpm-gpg/MY-CUSTOM-KEY",
+				},
+				{
+					Name:     "repo2",
+					BaseUrl:  "https://repo2.com",
+					Install:  false,
+					GPGCheck: false,
+					GPGKeys:  "file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY",
```

Comment:
Do we need `GPGKeys` if `GPGCheck` is set to `false`?

Reply:
No, but the default if blank will be our MS keys. In this case parsing the .json which doesn't set the keys will result in this output. (Open to dissenting opinions on how to handle that, but I'm looking to not break anyone using this. Currently we hard-code the keys to those two MS keys).